.github/workflows/tests.yml

```
@@ -0,0 +1,39 @@
+name: Tests
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node: [ '12.x', '14.x', '16.x' ]
```

current test target 10, 12 and 14 so we should probably stick to that
```suggestion
        node: [ '10.x', '12.x', '14.x' ]
```